plugin/pkg/admission/limitranger/admission.go

```diff
@@ -79,8 +86,27 @@ func (l *limitRanger) Admit(a admission.Attributes) (err error) {
 	if err != nil {
 		return admission.NewForbidden(a, fmt.Errorf("Unable to %s %v at this time because there was an error enforcing limit ranges", a.GetOperation(), a.GetResource()))
 	}
+
+	// if there are no items held in our indexer, check our live-lookup LRU, if that misses, do the live lookup to prime it.
 	if len(items) == 0 {
-		return nil
+		lruList, ok := l.liveLookupCache.Get(a.GetNamespace())
+		if !ok {
+			liveList, err := l.client.Core().LimitRanges(a.GetNamespace()).List(api.ListOptions{})
```

Comment:
isn't this prone to being wrong if the LRU cache is never cleared?  for example, if you add a limit range to a namespace, and later remove it, the LRU cache could have the wrong data?  also what happens if i delete and readd the same namespace, but the new namespace has no limit range?  this looks like it could return wrong data in that scenario?